rum-demo-app-ios/RUM Mobile Demo App/APIService/BaseRequestData.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BaseRequestData.swift
+//  RUM Mobile Demo App
+//
+//  Created by Akash Patel on 06/01/22.
+//
```

Comment:
For this and the following files, do we need these comments?

Reply:
This block of comment is autogenerated by Xcode whenever we create a swift file or header files. 
We can remove these line manually. 
Let me know if need to remove from all places.